openwisp_users/tests/test_admin.py

```diff
@@ -256,3 +256,24 @@ def test_admin_add_superuser_org_not_required(self):
         user = queryset.first()
         self.assertTrue(user.is_superuser)
         self.assertTrue(user.is_staff)
+
+    # def test_operator_change_user_permissions(self):
+    #     operator = self._create_operator()
+    #     operator.is_staff = True
+    #     # print(operator._meta.get_fields())
+    #     operator.user_permissions.set(Permission.objects.all())
+    #     self.client.force_login(operator)
+    #     admin = self._create_admin()
+    #     response = self.client.get(reverse('admin:openwisp_users_user_change', args=[admin.pk]))
+    #     html = '<input type="checkbox" name="is_superuser"'
```

Comment:
you should expect to not be able to reach that page, I think it redirects to the list of users
doing that would be good enough

Reply:
I can only check the `status_code` which is 302 so I can't check that `has_change_permission` function is redirecting to list of users.